packages/beta/src/cogniteClient.ts

```diff
@@ -55,6 +61,7 @@ export default class CogniteClient extends CogniteClientCleaned {
 
     this.httpClient.setDefaultHeader('cdf-version', 'beta');
 
+    this.simulatorsApi = this.apiFactory(SimulatorsAPI, 'simulators');
```

Comment:
is this API beta?

Reply:
You are right, it's alpha, we'll move it to alpha for now.